modules/base/src/properties/imageinformationproperty.cpp

```diff
@@ -98,6 +108,7 @@ void ImageInformationProperty::updateForNewImage(const Image& image) {
     }();
 
     dimensions_.set(dim);
+    aspectRatio_.set(static_cast<double>(dim.x) / dim.y);
```

Comment:
What if y is 0

Reply:
don't think that is valid.

Reply:
Potentially dimensions of 0 are not valid, but might never be checked.

Reply:
On the other hand, what should be shown in the property then? Right now it is a double property.